cmd/agent/dist/conf.d/postgres.d/conf_aws_aurora.yaml.default

```diff
@@ -0,0 +1,46 @@
+ad_identifiers:
+  - _dbm_aws_aurora
```

Comment:
You need to add `postgres` in the AD name, otherwise any DB will trigger postgres or mysql

```suggestion
  - _dbm_aws_aurora_pg
```